setup.py

```diff
@@ -8,7 +8,7 @@
 
 setup(
     name="nomic",
-    version="3.0.29",
+    version="3.0.30",
```

Comment:
i think this needs to be 3.0.31, can you merge from main?

Reply:
ah thanks for the catch, rebased and bumped